Day02_13_12_2023/Grouping.mjs

```diff
@@ -0,0 +1,110 @@
+class Grouping{
+    values = [];
+    header = [];
+
+    constructor(salesData){
+        this.values = salesData;
+        this.header = salesData.slice(1);
+    }
+
+    // Task 1 - A function is identifing the whether the product is poor, neutral, good.
+    categorizeProduct(columnName) {
```

Comment:
change the argument like this
```
[{
  group: "Poor",
   from:0,
  to: 100
}, 
.....
]
```

Reply:
Good Evening. I have solved the above issue. Please check once. Thank You.